src/lib/Microsoft.Health.Events/EventHubProcessor/IResumableEventProcessor.cs

```diff
@@ -11,7 +11,12 @@ namespace Microsoft.Health.Events.EventHubProcessor
 {
     public interface IResumableEventProcessor : IDisposable
     {
-        Task ResumeAsync(CancellationToken ct);
+        /// <summary>
+        /// Attempts to resume the Event Processor.
+        /// </summary>
+        /// <param name="ct">The cancellation token</param>
+        /// <returns><true> if the Event Processor was restart. <false> if it was already running.</returns>
+        Task<bool> ResumeAsync(CancellationToken ct);
```

Comment:
I think [ResumableAssignedPartitionProcessor.cs](https://github.com/microsoft/iomt-fhir/blob/658266857c920d8c14abd76c0eb5f16f5097975d/src/lib/Microsoft.Health.Events/EventHubProcessor/ResumableAssignedPartitionProcessor.cs#L29) also inherits from IResumableEventProcessor - do you mind adding the new bool return type for ResumeAsync? you can return false or something. I probably should have used a different interface

Reply:
Yea I just noticed that other class due to the build failure. Updated!